plugins/connection/httpapi.py

```diff
@@ -264,6 +275,7 @@ def send(self, path, data, retries=None, **kwargs):
             timeout=self.get_option("persistent_command_timeout"),
             validate_certs=self.get_option("validate_certs"),
             use_proxy=self.get_option("use_proxy"),
+            ciphers=self.get_option("ciphers"),
```

Comment:
If this gets passed all the time without checking, this will break httpapi for ansible-core<2.14 with the error `open_url() got an unexpected keyword argument 'ciphers'`

Reply:
Thank you for pointing this out. I have added version check in https://github.com/ansible-collections/ansible.netcommon/pull/494/commits/73e9010179949b941481b40fcc9af1af91a5b0d3 and tested with older versions locally.